<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>вуеусее</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}
QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="30,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>27</number>
      </property>
      <item row="17" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="17">
       <layout class="QHBoxLayout" name="image_layout"/>
      </item>
      <item row="0" column="0" rowspan="8">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>248</width>
            <height>501</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="nano960_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>nano960 9 classes</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="nano_960_10">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>nano960 10 classes</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="medium_960_10">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>medium960 10 classes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="nano_1280_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>nano1280 9 classes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="medium_960_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>medium960 9 classes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="medium_ships">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ships</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="slider_lbl">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Порог уверенности: 50</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="conf_slider">
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #bbb;
}

/*1. Sliding sliding slot design parameters*/
QSlider::sub-page:horizontal {
     /*Slot color*/
    background: rgb(70,130,180);
     /*Found angle of the outer ring area*/
    border-radius: 2px;
     /*Cover the area height*/
    margin-top:8px;
     /*Cover the area height*/
    margin-bottom:8px;
}

/*2. Unexpectedly sliding slot design parameters*/
QSlider::add-page:horizontal {
    /*Slot color*/
    background: rgb(105,105, 105);
   
    border: 1px solid #777;
    /*Found angle of the outer ring area*/
    border-radius: 2px;
     /*Cover the area height*/
    margin-top:8px;
     /*Cover the area height*/
    margin-bottom:8px;
}

/*3. Usually sliding slider design parameters*/
QSlider::handle:horizontal {
    /*Slider color*/
    background: rgb(193,204,208);
    /*The width of the slider*/
    width:40px;

    /*Slider outer ring is 1px, add color*/
    border: 1px solid rgb(193,204,208);

     /*Sliding block outer ring upside -round angle*/
    border-radius: 15px;
     /*Cover the area height*/
    margin-top:2px;
     /*Cover the area height*/
    margin-bottom:2px;
}

/*4. The slider design parameters displayed during manual pull*/
QSlider::handle:horizontal:hover {
    /*Slider color*/
    background: #484848;
    /*The width of the slider*/
    width: 40px;
    
    border: 1px solid #484848;
     /*Sliding block outer ring upside -round angle*/
    border-radius: 15px;
     /*Cover the area height*/
    margin-top:2px;
     /*Cover the area height*/
    }</string>
             </property>
             <property name="maximum">
              <number>99</number>
             </property>
             <property name="pageStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="sliderPosition">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="no_detected">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>no_detected</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="detected">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>detected</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="open_btn">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"> QPushButton#pbReady {
    padding:4px;
    color: #fff;
    font-size: 14px;
    border-radius: 2px;
    border: 1px solid #3873d9;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #4287ff, stop: 1 #356ccc);
 }
QPushButton:pressed {
    color: #111;
    border: 1px solid #3873d9;
    background: #fff;
 }</string>
               </property>
               <property name="text">
                <string>open</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="detect_btn">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>detect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="save_btn">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>save</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1515</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
